dan_layer/wallet/sdk/src/storage.rs

```diff
@@ -128,6 +128,10 @@ pub trait WalletStoreReader {
         account_name: &str,
         status: OutputStatus,
     ) -> Result<Vec<ConfidentialOutput>, WalletStorageError>;
+    fn get_proof_by_transaction_hash(
```

Comment:
nit: prefer the naming scheme is followed `{entity}_{action}`
```suggestion
    fn proofs_get_by_transaction_hash(
```